<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ORCAMDlg</class>
 <widget class="QDialog" name="ORCAMDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ORCAM</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,1,1">
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="modelView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineFilter"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnFindNext">
     <property name="text">
      <string>Find next</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>234</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
